public/data/TusharAgarwal18.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Tushar Agarwal",
+  "type": "Personal",
+  "bio": "Programmer",
+  "avatar": "https://github.com/TusharAgarwal18.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/TusharAgarwal18",
+      "icon": "github"
+    },
+    {
+      "name": "",
+      "url": "",
+      "icon": ""
+    }
+  ],
+  "milestones": [
+    {
+      "title": "",
+      "date": "",
+      "icon": "",
+      "color": "",
+      "description": "",
+      "url": ""
+    }
```

Comment:
I recommend removing these empty fields if you're not going to use them.

Reply:
Thanks